packages/app/src/pages/EditorRoomPage.tsx

```
@@ -234,30 +267,61 @@ export const EditorRoomPage = () => {
+                          if (confirm("Delete draft?")) removeDraft(d.id);
+                        }}
+                      >
+                        Delete
+                      </Button>
                     </div>
                   </div>
                   <div className="mt-2 text-[15px] leading-relaxed text-text-primary">
-                    <div className="prose max-w-none overflow-hidden max-h-24" dangerouslySetInnerHTML={{ __html: renderMarkdownToHtml(d.content || "") }} />
+                    <div
+                      className="prose max-w-none overflow-hidden max-h-24"
+                      dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)